run.py

```diff
@@ -905,6 +906,99 @@ def hvreads(args):
             "aws", "s3", "cp", output, "%s/%s/hvreads/%s" % (
                S3_BUCKET, args.bioproject, output)])
 
+def alignments(args):
+   available_inputs = get_files(args, "hvreads")
+   existing_outputs = get_files(args, "alignments")
+
+   for sample in get_samples(args):
+      output = "%s.alignments.tsv" % sample
+      if output in existing_outputs: continue
+
+      input_fname = "%s.hvreads.json" % sample
+      if input_fname not in available_inputs: continue
+
+      with tempdir("alignments", input_fname) as workdir:
+         subprocess.check_call([
+            "aws", "s3", "cp", "%s/%s/hvreads/%s" % (
+               S3_BUCKET, args.bioproject, input_fname), input_fname])
+
+         pair1 = []
+         pair2 = []
+         combined = []
+         with open(input_fname) as inf:
+            for seq_id, details in sorted(json.load(inf).items()):
+               if type(details[0]) == type(1):
+                  assignment = details.pop(0)
+               kraken_info, *reads = details
+               assert len(reads) in [1, 2]
+               if len(reads) == 1:
+                  read, = reads
+                  combined.append((seq_id, read))
+               else:
+                  read1, read2 = reads
+                  pair1.append((seq_id, read1))
+                  pair2.append((seq_id, read2))
+
+         for label, reads in [
+               ("pair1", pair1),
+               ("pair2", pair2),
+               ("combined", combined),
+         ]:
+            if reads:
+               with open("%s.fastq" % label, "w") as outf:
+                  for seq_id, read in reads:
+                     seq, quality = read
+                     outf.write("@%s\n%s\n+\n%s\n" % (seq_id, seq, quality))
```

Comment:
Does it make sense to use biopython to do this? I'm not sure where the current format makes assumptions about quality score notations. 

```
with open(f"{label}.fastq", "w") as outf:
            seq_records = []
            for seq_id, read in reads:
                seq, quality = read
                seq_record = SeqRecord(Seq(seq), id=seq_id, description="", letter_annotations={"phred_quality": quality})
                seq_records.append(seq_record)
            SeqIO.write(seq_records, outf, "fastq")
```

Reply:
I don't think using `SeqRecord` really gets us anything here: the fastq format is simple enough to write, and the quality scores we're writing came from other fastqs.

I *do* think we should use standard tools (`FastqGeneralIterator` and `SimpleFastaParser`) for parsing, though.  The formats are complex enough that you can have subtle bugs there.